components/Backgrounds/Basic-BG.js

```diff
@@ -19,4 +19,60 @@ const StyledMain = styled.main`
   z-index: -1;
 `;
 
+export const StyledLevel1 = styled.main`
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  height: auto;
+  width: auto;
+  max-width: 100%;
+  background-image: url("/assets/ali-abdul-rahman-Xva-TYqwHhA-unsplash.jpg");
+  background-size: cover;
+  z-index: -1;
+`;
+
+export const StyledLevel2 = styled.main`
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  height: auto;
+  width: auto;
+  max-width: 100%;
+  background-image: url("/assets/angus-gray-qEaELLSYZW0-unsplash.jpg");
+  background-size: cover;
+  z-index: -1;
+`;
+
+export const StyledLevel3 = styled.main`
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  height: auto;
+  width: auto;
+  max-width: 100%;
+  background-image: url("/assets/silas-baisch-f4bfvVXKZP8-unsplash.jpg");
+  background-size: cover;
+  z-index: -1;
+`;
+
+export const StyledLevel0 = styled.main`
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  height: auto;
+  width: auto;
+  max-width: 100%;
+  background-image: url("/assets/shifaaz-shamoon-okVXy9tG3KY-unsplash.jpg");
+  background-size: cover;
+  z-index: -1;
+`;
+
```

Comment:
the only difference of these styles is the url of the background image.
Please create a more generic styled component that gets the background-image passed as prop (or even better: the level passed as prop and you get the url out of an array)